packages/foreman/rubygem-rkerberos/rubygem-rkerberos.spec

```diff
@@ -0,0 +1,98 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name rkerberos
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.5
+Release: 2%{?dist}
+Summary: A Ruby interface for the the Kerberos library
+Group: Development/Languages
+License: Artistic 2.0
+URL: https://github.com/domcleal/rkerberos
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
```

Comment:
Fedora carries a [patch](https://src.fedoraproject.org/rpms/rubygem-rkerberos/blob/master/f/rkerberos-0.1.5-gcc8-argnum-fix.patch) for compilation with gcc8 and ruby 2.5 -- might be needed when we go for CentOS8. Not sure if we should just pull it in now.

Reply:
Let's cross that bridge given I am not sure what version of things we will be on when we arrive there.